src/ngraph/op/fused/elu.hpp

```diff
@@ -35,13 +35,16 @@ namespace ngraph
             ///
             /// \param data Input tensor
             /// \param alpha Multiplier for negative values
-            Elu(const std::shared_ptr<ngraph::Node>& data,
-                const std::shared_ptr<ngraph::Node>& alpha);
+            Elu(const std::shared_ptr<ngraph::Node>& data, const double& alpha);
```

Comment:
PODs should not be passed by reference, please change to a simple `const double` for the `alpha` param.